src/OTSubscriber.js

```diff
@@ -81,18 +81,22 @@ export default class OTSubscriber extends Component {
   }
   render() {
     const containerStyle = this.props.containerStyle;
```

Comment:
`containerStyle` can be within the first if block.

Reply:
Done. I will just update the docs now.